how-to.md

```diff
@@ -1,8 +1,48 @@
 # How to
 
+## Add new MicroProfile Implementation
+
+What are the technical steps to add a new MicroProfile Implementation?
```

Comment:
I suggest not to have this line as it does not add anything more. We have headings already.